common/app/templates/inlineJS/blocking/cloudwatchBeacons.scala.js

```diff
@@ -1,2 +1,22 @@
 @(item: model.MetaData)(implicit request: RequestHeader)
 @import conf.switches.Switches._
+
+@if(OfflinePageView.isSwitchedOn) {
+    (function () {
+        try {
+            var cookies = document.cookie.split(';');
+            var offlineViews = 0;
+            for (var len = cookies.length, i = 0; i < len ; i++) {
+                var data = cookies[i].split('=');
+                if (data[0].indexOf('gu.offlineViews') !== -1) {
+                    offlineViews = parseInt(data[1]);
+                    break;
+                }
+            }
+            for (var i = 0; i < offlineViews; i++) {
+                (new Image()).src = window.guardian.config.page.beaconUrl + '/count/offline-page-view.gif';
+            }
+            document.cookie = 'gu.offlineViews=0;Max-Age=0';
```

Comment:
> Ahh is it because you can't access local storage from the SW?

we're doing it from the page to get over the SW/sync api gulf, so it could work. the protocol was the issue